core/src/main/java/com/linecorp/armeria/client/endpoint/dns/DnsAddressEndpointGroupBuilder.java

```diff
@@ -253,4 +253,15 @@ public DnsAddressEndpointGroupBuilder selectionTimeout(Duration selectionTimeout
     public DnsAddressEndpointGroupBuilder selectionTimeoutMillis(long selectionTimeoutMillis) {
         return (DnsAddressEndpointGroupBuilder) super.selectionTimeoutMillis(selectionTimeoutMillis);
     }
+
+    @Override
+    public DnsAddressEndpointGroupBuilder addDnsQuestionListeners(
+            Iterable<? extends DnsQuestionListener> dnsQuestionListeners) {
+        return (DnsAddressEndpointGroupBuilder) super.addDnsQuestionListeners(dnsQuestionListeners);
+    }
+
+    @Override
+    public DnsAddressEndpointGroupBuilder addDnsQuestionListener(DnsQuestionListener dnsQuestionListener) {
```

Comment:
- Could use a plural form with varargs?
- This class is already about DNS, how about simplifying the names a little bit, e.g. `addQueryListener()`?
  - `DnsQuestionListener` could be renamed to `DnsQueryListener`